sentences/fr/_common.yaml

```diff
@@ -319,7 +319,9 @@ expansion_rules:
   yatil: "(y a[-][ ]t[-][']il|il y a)"
   estil: "(est|sont)[-][ ][(il[s]|elle[s])]"
   atil: "(ont|a)[-][ ][t][ ][-][(il[s]|elle[s])]"
-  tous: "(tous|toutes)"
+  quel: "quel[le][s]"
+  capteur: "(capteur|appareil|machine|chose|sonde)[s]"
```

Comment:
- `chose` is too generic
- `appareil` is the translation of `device`, not `sensor`
- `machine` is more a synonym of `device`, but I can't find an appropriate sentence where it can be used